tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
+Your VM is ready for you to install and run sample programs from the Kite SDK Examples project.
+
+## Installing the VM and Kite Examples
+
+1. Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
+1. In that VM, run the following command from a terminal window. This command clones a local copy of Kite code examples you can build and run.
+
+     ```bash
+     git clone https://github.com/kite-sdk/kite-examples.git
+     ```
+
+1. If you haven't already done so, install the [Kite CLI][install-cli].
+
+[getvm]: http://www.cloudera.com/content/support/en/downloads/quickstart_vms.html
+[install-cli]:{{site.baseurl}}/Install-Kite.html
+
+## Configuring the VM
+
+Some Kite examples require Flume. If you use Cloudera Manager, Flume user impersonation is configured for you. If do not use Cloudera Manager, you must update Flume user impersonation in `core-site.xml`.
```

Comment:
I think we need to add more background on Cloudera Manager. The steps above get you a working VM with Kite, but don't mention CM, while this assumes the user knows about the choice and has already decided whether to use CM. I think a small section to introduce that choice is needed. It should include the fact that we recommend using CM, how to start it (opening the link on the desktop, I believe), and why you would choose not to use it (it requires more memory allocated to the VM).